frontend/src/components/LoginModal.jsx

```diff
@@ -31,33 +21,57 @@ function LoginModal({ modalVisible, closeModal }) {
 	const handleSubmit = async (event) => {
 		event.preventDefault();
 
-		//**Not needed yet */
-		// const response = await loginUser({
-		//   username,
-		//   password
-		// });
+		if (userName.length < 5) {
```

Comment:
Sorry, Miscommunication upon the required validation feature. **Username and password validation** should be present upon signup/changing username or password. You have changed the `patch` endpoint already, but the change in the `login-modal` is unnecessary